code/game/machinery/computer/buildandrepair.dm

```diff
@@ -422,6 +422,21 @@
 	var/state = STATE_EMPTY
 	var/obj/item/circuitboard/circuit = null
 
+/obj/structure/computerframe/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>It is <b>[anchored ? "bolted to the floor" : "unbolted"]</b>. [anchored ? "" : "Alt-Click to rotate it."]</span>"
+	switch(state)
+		if(STATE_EMPTY)
+			. += "<span class='notice'>The frame is <b>welded together</b>, but it is missing a <i>circuit board</i>.</span>"
+		if(STATE_CIRCUIT)
+			. += "<span class='notice'>A circuit board is <b>firmly connected</b>, but the cover is <i>unscrewed and open</i>.</span>"
+		if(STATE_NOWIRES)
+			. += "<span class='notice'>The cover is <b>screwed shut</b>, but the frame is missing <i>wiring</i>.</span>"
```

Comment:
```suggestion
			. += "<span class='notice'>The cover is <b>screwed</b> shut and the frame is missing its <i>wiring</i>.</span>"
```
Better to have these kind of things be one whole sentence instead of a `, but` statement imo

Reply:
I'm doing the same thing as the airlock frames, and imo `The cover is screwed shut and the frame is missing wiring.` doesn't work as well.